admin/grants.sql

```diff
@@ -281,15 +285,16 @@ GRANT
     USAGE
 ON FUTURE SCHEMAS IN DATABASE SYNAPSE_DATA_WAREHOUSE_DEV
 TO ROLE DATA_ENGINEER;
-GRANT INSERT, SELECT, UPDATE ON FUTURE TABLES IN DATABASE SYNAPSE_DATA_WAREHOUSE_DEV
+GRANT INSERT, SELECT, UPDATE, EVOLVE SCHEMA ON FUTURE TABLES IN DATABASE SYNAPSE_DATA_WAREHOUSE_DEV
```

Comment:
I don't see `MODIFY` or `DELETE` mentioned here. Would we still be able to for example test adding columns to a table in a test database we created and also delete that?

Reply:
`MODIFY` isn't an allowed permission and i'll hold off on `DELETE` and `TRUNCATE`